hmpy/views/progress_bar.py

```
@@ -0,0 +1,316 @@
+from .view import View
+from PyQt5.QtGui import QPainter, QPen, QColor, QFont, QFontMetrics
+from PyQt5.QtWidgets import QProgressBar
+from PyQt5.QtCore import Qt, QRect
+from enum import IntEnum
+
+
+class ProgressBarView(View):
+    """View simulating a Gauge."""
+
+    _LINE_START_Y = 55
+    _LINE_START_X = 10
+    _MARK_SCALE = 80
+    _SCALE = 25
+    _PROGRESS_SCALE = 1600
+
+    class Orientation(IntEnum):
+        HORIZONTAL = 0
+        VERTICAL = 1
+
+    def __init__(self, title, value=0, unit_text="%",
```

IMO title can be optional, probably just default to ""

---

Defaulting unit text to % may be misleading, since we arent actually displaying the perccent progress.  For example if I set the max to 125, when the bar is full it would display 125%, but I would expect it to show 100%